### ISSUE TITLE
Running `bash` as a shell command freezes Femto-Emacs

### ISSUE THREAD

**Issue by siraben** [FemtoEmacs/Femto-Emacs#11]
After performing `C-x @` and type `bash RET`, Femto-Emacs freezes and the process must be killed externally.

**Comment by hughbarney** [comment]
Agreed this is a bug.  Note that starting a new shell was not what this command is designed for. The idea is to send a command to the shell and collect the output to a temporary file.  For example 'ls -l' will run the command ls -l in the shell and the output will appear in a window.  The issue could be fixed by checking if the command arguments are executing a shell.  It looks like MicroEmacs checks if a shell is being run and falls back to simply dropping out to a shell until you type 'exit'.  If you try this on Zile it will just return an empty window.   Will fix when I get time in the femto repository at: https://github.com/hughbarney/femto
